server/utils/url-utils.js

```diff
@@ -2,7 +2,7 @@ const isDev = process.env.NODE_ENV !== 'production';
 const isBeta = !!process.env.BETA;
 
 export function getEmailSender() {
-  return process.env.EMAIL_SENDER || 'team@freecodecamp.org';
+  return process.env.SES_MAIL_FROM || 'team@freecodecamp.org';
```

Comment:
Updating this var as per https://github.com/freeCodeCamp/freeCodeCamp/issues/15838#issuecomment-327030377 because we would still need a var to allow MAIL FROM for anyone using SES.